circlemator.gemspec

```diff
@@ -1,41 +1,42 @@
 # frozen_string_literal: true
-lib = File.expand_path('../lib', __FILE__)
+
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'circlemator/version'
+require "circlemator/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'circlemator'
+  spec.name          = "circlemator"
   spec.version       = Circlemator::VERSION
-  spec.authors       = ['Emanuel Evans']
-  spec.email         = ['emanuel@rainforestqa.com']
+  spec.authors       = ["Emanuel Evans"]
+  spec.email         = ["emanuel@rainforestqa.com"]
 
-  spec.summary       = 'A bucket of tricks for CircleCI and Github.'
+  spec.summary       = "A bucket of tricks for CircleCI and Github."
   spec.description   = <<-EOF.strip
     A few utilities for CircleCI to improve your CI workflow.
   EOF
-  spec.homepage      = 'https://github.com/rainforestapp/circlemator'
-  spec.license       = 'MIT'
+  spec.homepage      = "https://github.com/rainforestapp/circlemator"
+  spec.license       = "MIT"
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = 'exe'
+  spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ['lib']
+  spec.require_paths = ["lib"]
 
-  spec.add_dependency 'httparty', '>= 0.13.7', '< 0.19.0'
-  spec.add_dependency 'pronto', '~> 0.11.0'
-  spec.add_dependency 'pronto-rubocop', '~> 0.11.0'
-  spec.add_dependency 'pronto-commentator', '~> 0'
-  spec.add_dependency 'pronto-undercover', '~> 0.1'
-  spec.add_dependency 'pronto-brakeman', '~> 0.11.0'
+  spec.add_dependency "httparty", ">= 0.13.7", "< 0.19.0"
+  spec.add_dependency "pronto", "~> 0.11.1"
+  spec.add_dependency "pronto-rubocop", "~> 0.11.4"
+  spec.add_dependency "pronto-commentator", "~> 0"
+  spec.add_dependency "pronto-undercover", "~> 0.1"
+  spec.add_dependency "pronto-brakeman", "~> 0.11.1"
```

Comment:
Had to update some of these `pronto` versions to be able to upgrade to Ruby 3.2.0.